.circleci/config.yml

```diff
@@ -7,11 +7,10 @@ defaults: &defaults
     xcode: "11.4.0"
   working_directory: ~/amplify-ios
   environment:
-      BUNDLE_PATH: vendor/bundle
+    BUNDLE_PATH: vendor/bundle
 
 references:
-  repo_cache_key: &repo_cache_key
-    v1-repo-{{ .Branch }}-{{ .Revision }}
+  repo_cache_key: &repo_cache_key v1-repo-{{ .Branch }}-{{ .Revision }}
```

Comment:
Will this work? My vague recollection of yaml is that the alias thing needs to be on its own line.

Reply:
VS Code kindly formatted that file for me, hence the large number of superficial changes (:

But it will work. I validated the config with `circleci config validate`, which checks ensures that the yaml in the config is sound.